_posts/2020-06-26-Making-IrishGen-SPARQL-Part-I.md

```diff
@@ -0,0 +1,549 @@
+---
+layout: post
+date: 2020-06-26
+title: "Making IrishGen SPARQL Part I: Selects"
+author: cgy
+---
+
+{% assign author = site.data.authors[page.author]%}
+{{author.name}}
+
+ORCID: <a href="https://orcid.org/{{ author.orcid }}" title="{{author.name}}">{{author.orcid}}</a>
+
+This post will begin to cover the [SPARQL query
+language](https://www.w3.org/TR/sparql11-overview/) using IrishGen as
+the example dataset.  This is the culmination of several posts that
+guide the reader through the sometimes confusing world of Linked Data.
+The post assumes that the reader is already familiar with [Linked
+Data, Semantic Web]({% post_url
+2020-06-17-IrishGen-RDF-Linked-Data-Semantic-Web %}), [Triplestores,
+and logical reasoning using OWL 2](% post_url
+2020-06-22-Triplestores-Onotologies-Reasoning %}).  [Eystein
+Thanisch](https://orcid.org/0000-0003-2819-5519) has already covered
+some of this in [his post]({% post_url
+2020-06-20-Some-Examples-of-Querying %}) which gives a practical and
+useful introduction to using SPARQL.  This and following posts is
+meant to deepen the reader's understanding and to give examples of
+most forms of SPARQL so that the reader feels confident in asking and
+answering their own questions of the IrishGen dataset.
+
+## Baeth: A Useful Idiot
```

Comment:
Are we sure it is to be translated "idiot"? Maybe it was a name given to someone with what would today be termed a learning disability, in which case this is a pretty inappropriate spin to put on that. Do you have any examples of it being used as an insult?

Maybe add some comment at the beginning, where you give the dictionary definition, saying that we don't know why it was used as a personal name but consider the various possibilities (someone who is actually an idiot, someone who is mentally incapable for medical reasons, an innocent/holy fool?).

Reply:
Yeah, I was trying to be cutesy and it's not a good idea so I will change it.